src/quote/quote.jsx

```diff
@@ -1,64 +1,62 @@
 import React, { Component } from 'react';
+import {fetchQuotes} from '../api';
 import './styles.css'
-import $ from 'jquery';
-export default class BlockQuote extends React.Component {
+class BlockQuote extends Component {
```

Comment:
- Rename this component to ViewQuotesPage
- Make a presentational Component Named `Quote` and import that component as a child  whoch will be a dumb component